.infra/index.ts

```diff
@@ -260,6 +285,15 @@ const [apps] = deployApplicationSuite(
         },
       ],
     },
+    additionalSecrets: [
+      {
+        name: 'cert-secret',
+        data: {
+          'public.pem': Buffer.from(cert.public).toString('base64'),
+          'key.pem': Buffer.from(cert.key).toString('base64'),
+        },
+      },
+    ],
```

Comment:
It's a 2048 RSA key

Reply:
Yeah, my bad. I misremembered the `kubernetes.io/tls` secret type. Although it could be used here as well. It does not really matter if it is `opaque` or `kubernetes.io/tls`, and could be confusing if it is the latter when it is not used for TLS.